instrumentation/tests/dotnet/test.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -euox pipefail
+
+arch="${ARCH:-amd64}"
```

Comment:
`amd64` wouldn't work for `dotnet --architecture` but per previous comment, you don't need to pass that.

Reply:
Related Q.: is this `arch` to facilitate `arm64` builds?

Reply:
yes, the tests run both amd64 and arm, for Java, node, and .NET now.